### PR TITLE
[ADD] accounting: add Vietnam localisations

### PR DESCRIPTION
**Pull request by tong-odoo**
Impacted Version:

- 16.4 and above

This commit improve below features:

- Vietnam accounting documentation
- Vietnamese QR banking code documentation

task-3344654

### CODE REVIEW

File: content/applications/finance/fiscal_localizations/vietnam.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Go to :menuselection:`Accounting --> Configuration --> Settings`, and under the :guilabel:`Customer
+Payments` section, activate the :guilabel:`QR Codes` feature.
+
+QR codes bank account configuration
+-----------------------------------
+
+Go to :menuselection:`Contacts --> Configuration --> Bank Accounts` and select the bank account for
+which you want to activate **Vietnamese QR banking**. Set the :guilabel:`Bank Identifier Code` on
+the bank, then set the :guilabel:`Proxy Type` and fill in the :guilabel:`Proxy Value` field
```
Comment (toaa-odoo) [inline content/applications/finance/fiscal_localizations/vietnam.rst:23]:
```suggestion
the bank, then set the :guilabel:`Proxy Type` and fill in the :guilabel:`Proxy Value` field
```
proxy

I couldn't find the "Proxy Type" field on the contact form. Is it still relevant ? :)

**Review by toaa-odoo — CHANGES_REQUESTED** [review]
Hi @tong-odoo ! Thanks for your work, I made minor phrasing changes, but I couldn't find one option described in the doc. Let me know where I can find it, if still applicable :)

File: content/applications/finance/fiscal_localizations/vietnam.rst (hunk 1 above)
Comment (tong-odoo) [reply content/applications/finance/fiscal_localizations/vietnam.rst:23]: Hello @toaa-odoo, May I ask which version are you using? This seems like an older version of the development

Comment (tong-odoo) [reply content/applications/finance/fiscal_localizations/vietnam.rst:23]: The proxy type is still required.

Comment (tong-odoo) [reply content/applications/finance/fiscal_localizations/vietnam.rst:23]: @toaa-odoo You could try this runbot: https://47551880-master-all.runbot104.odoo.com/

Comment (toaa-odoo) [reply content/applications/finance/fiscal_localizations/vietnam.rst:23]: I had used the one linked in the chatter of the task. The one you just provided work though! All good from me, I'll update @dade-odoo 's suggestion

**Review by toaa-odoo — APPROVED** [review]

**Review by jcs-odoo — APPROVED** [review]
@robodoo delegate+

**Comment by tong-odoo** [comment]
@robodoo r+

**Comment by tong-odoo** [comment]
@robodoo r+

**Pull request closed without merging**